flycheck.el

```diff
@@ -2639,7 +2689,7 @@ WORKING-DIR."
         (flycheck-error-list-refresh)
         (run-hooks 'flycheck-after-syntax-check-hook)
         (when (eq (current-buffer) (window-buffer))
-          (flycheck-display-error-at-point))
+          (flycheck-automatically-display-error-at-point))
```

Comment:
I would rather add a check for `flycheck-enable-display-error-at-point` in the `when`, and get rid of the new function.